bazel/src/test/data/googleapis/google/example/library/v1/BUILD.bazel.updated.baseline

```diff
@@ -104,10 +93,9 @@ java_gapic_test(
 java_gapic_assembly_gradle_pkg(
     name = "renamed_java_rule",
     include_samples = True,
-    transport = "grpc+rest",
+    transport = "rest",
```

Comment:
Are we retaining test cases with `grpc+rest`?  It's the expected most common usage, so we should.

Reply:
Which test cases do you mean? If you mean "a test case to retain grpc+rest across the file when **re**generated" then we don't have one of those today. I can add one though!

This test case was actually wrong because the `java_gapic_library` and `java_gapic_assembly_gradle_pkg` did not have the same value - the former was `lightning` and this was `grpc+rest` - they should always match.

Reply:
Might be worth adding that test case, but that can come in a separate PR.